src/utils/camelCaseKeys.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { Inbox } from '@/types/Inbox';
+import { camelCase } from './camelCase';
+
+export const transformInbox = (inbox: any): Inbox => {
```

Comment:
I believe this shouldn’t be a separate helper file; it should be integrated into the apiService. It shouldn’t be used outside of the service.

Reply:
@pranavrajs Makes sense. Currently, it's only used in the Inbox store, but we're using the transform methods in other services like Conversation and Message as well. For example, we need to use transformConversation whenever a new conversation is created via action cable event (https://github.com/chatwoot/chatwoot-mobile-app/blob/78b06bd29f4ded4717b39577d6981cb17f442413/src/utils/actionCable.ts#L63). That's why I thought of creating a common utility that we can use anywhere and add the necessary test cases.